maps/site53/site53-1.dmm

```diff
@@ -2631,8 +2631,13 @@
 	dir = 1
 	},
 /obj/structure/table/standard,
+/obj/item/device/scanner/plant,
+/obj/item/device/scanner/plant,
+/obj/item/device/scanner/plant,
+/obj/item/device/scanner/plant,
+/obj/item/device/scanner/plant,
 /turf/simulated/floor/tiled/monotile/white,
-/area/site53/llcz/dclass/botany)
+/area)
```

Comment:
Did you accidentally mess up the area? I'm no mapper, that's harry's job, but this looks odd

Reply:
I certainly hope not but would not be too surprised if I did somehow do this.

Reply:
Oh I know what this is. I accidentaly deleted a tile I believe and replaced it, but forgot to replace the area. One moment.

Reply:
.. Now i'm confused. Tried doing it again and made sure I only placed down objects without messing up the area, but it's doing the same thing regardless. So I have to assume it's intended..?

Reply:
It got updated, no worries